vioscsi/vioscsi.c

```diff
@@ -394,7 +406,8 @@ DriverEntry(
     hwInitData.NumberOfAccessRanges     = PCI_TYPE0_ADDRESSES;
     hwInitData.MapBuffers               = STOR_MAP_NON_READ_WRITE_BUFFERS;
 
-    hwInitData.SrbTypeFlags = SRB_TYPE_FLAG_STORAGE_REQUEST_BLOCK;
+    hwInitData.SrbTypeFlags             = SRB_TYPE_FLAG_STORAGE_REQUEST_BLOCK;
+    hwInitData.AddressTypeFlags         = ADDRESS_TYPE_FLAG_BTL8;
```

Comment:
Not related to trace.

Reply:
No.
IIRC, this was introduced in Win8.1.
I would have added this to my WIP when I added all missing init options. Would you prefer it in a separate PR?

Reply:
Yes, please.

Reply:
Done..!